components/BottomNavigation/src/MDCBottomNavigationBar.m

```diff
@@ -547,7 +547,13 @@ - (void)setItems:(NSArray<UITabBarItem *> *)items {
           key, kMaterialBottomNavigationStringsTableName, [[self class] bundle], kOfString);
       NSString *localizedPosition =
           [NSString localizedStringWithFormat:itemOfTotalString, (i + 1), (int)items.count];
-      itemView.button.accessibilityHint = localizedPosition;
+      // Allow a custom `accessibilityHint` to be assigned even if "faking" a tab bar is enabled.
+      if (itemView.button.accessibilityHint.length) {
+        itemView.button.accessibilityHint =
+            [NSString stringWithFormat:@"%@. %@", localizedPosition, itemView.accessibilityHint];
```

Comment:
Should this be localized?

Reply:
No, I don't think so.  The `accessibilityHint` could be first in ordering, but it's not a matter of localization. In other words, Hebrew speakers wouldn't expect the `accessibilityHint` to come at a different position than the "faked" positional data.

Reply:
I chose the `accessibilityHint` to come last in this case since it typically comes last on accessibility elements.